src/components/pages/auth/register/index.tsx

```diff
@@ -153,7 +156,9 @@ export default function RegisterComponent() {
                 className="w-full py-2 text-sm md:text-base rounded-full text-white bg-primary transition duration-300 ease hover:bg-secondary"
                 disabled={registerStatus === "success" || registerStatus === "registering"}
               >
-                Register Now
+                {
+                isLoading ? <Loader size={24} className="mx-auto animate-spin self-center"/> : <p>Register Now</p>
```

Comment:
Just a question, is self-center working here since it is used with flex class

Reply:
nope I'll correct it. !